compose.yml

```diff
@@ -27,6 +28,14 @@ services:
             - 'traefik.enable=true'
             - 'traefik.http.routers.${DOCKER_CONTAINER_NAME:-transmorpher}.rule=Host(`${DOCKER_CONTAINER_DOMAIN:-transmorpher.test}`)'
             - 'traefik.http.services.${DOCKER_CONTAINER_NAME:-transmorpher}.loadbalancer.server.port=80'
+        # runtime: nvidia
+        # deploy:
+        #   resources:
+        #     reservations:
+        #       devices:
+        #         - driver: nvidia
+        #           count: all
+        #           capabilities: [ gpu,video,utility,compute ]
```

Comment:
this forces people or systems into changing a versioned file, which should not be committed with gpu support activated, as it will break the github pipelines otherwise

Reply:
this has been moved to an optional yml file